src/audio-manager.ts

```diff
@@ -0,0 +1,208 @@
+import {_audioContext} from './audio-listener.js';
+
+const _bufferCache: Map<string, [AudioBuffer, number]> = new Map();
+
+function _remove(source: string) {
+    if (_bufferCache.has(source)) {
+        const [, referenceCount] = _bufferCache.get(source)!;
+        if (referenceCount > 1) {
+            const [audioBuffer, referenceCount] = _bufferCache.get(source)!;
+            _bufferCache.set(source, [audioBuffer, referenceCount - 1]);
+        } else {
+            _bufferCache.delete(source);
+        }
+    }
+}
+
+/** AudioManager loads and manages audiofiles from which PlayableNodes are created
+ * @example
+ * ```
+ * async start() {
+ *      this.audio = await AudioManager.load('click.wav');
+ * }
+ * onPress() {
+ *      this.audio.play();
+ * }
+ * // if not needed any longer
+ * this.audio.destroy();
+ * ```
+ */
+export const AudioManager = {
```

Comment:
You can make a class actually, and instantiate it. I don't know now if an object makes sense at all. Maybe `load` should just be the default export 🤔

Reply:
Yes, i was wondering myself if an object was necesarry. But i still want the user to know somehow that the load function is part of the audio manage and not the rest of the spatial-audio stuff...